basics/components/schema.md

```diff
@@ -91,6 +91,8 @@ Data type helps in determining the operations which can be performed on the colu
 `BOOLEAN`, `TIMESTAMP`, `JSON` are added after release `0.7.1`. In release `0.7.1` and older releases, `BOOLEAN` is equivalent to `STRING.`
 {% endhint %}
 
+Pinot also supports columns that contain lists or arrays of items, but there isn't an explicit data type for these columns. Instead, you can indicate that a dimension column accepts multiple values. For more information, see [DimensionFieldSpec](https://docs.pinot.apache.org/configuration-reference/schema#dimensionfieldspec) in the Schema configuration reference.
```

Comment:
s/explicit data type for these columns/explicit data type to represent these lists or arrays.

Reply:
I actually did the changes on this PR via gitbook, so I've now applied your suggestion via gitbook as well!